.github/workflows/python-package.yml

```diff
@@ -0,0 +1,35 @@
+name: Python package
+
+on: [push]
+
+env:
+  MIN_CODE_COVERAGE_PERCENT: 93
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install Python
+        uses: actions/setup-python@v4
+        with:
+          python-version-file: '.python-version'
```

Comment:
Since I cannot find any other usage of this file elsewhere in the repo, I'd suggest making the python version an environment variable here in the workflow and using it from there.

Reply:
I use `pyenv` to manage the python version of the projects I am working on. This `.python-version` file acts as a single source of truth for dev and CI environments. So I will keep this file as it is.

Do you think the README.md file should mention something about this file and pyenv? IF so, how would you phrase it?

Reply:
Ah I understand, thanks!

To be honest, I don't quite see why this file would be added to the public codebase as it's merely an artifact of personal preference of setting up a local env. `pyproject.toml` specifies which python versions are supported and developers may choose different ways to make their local env compatible with it. Anyway, it's also more nit-picking than an actual needed change, I think. As you said, adding a line or two to the `README.md` that states why it's there and how to use it with pyenv would help as well. E.g. a small sub-section in the "Contributions" sections.